fsw/src/sample_app.c

```diff
@@ -224,26 +399,70 @@ void SAMPLE_ReportHousekeeping(void)
 /*         part of the task telemetry.                                        */
 /*                                                                            */
 /* * * * * * * * * * * * * * * * * * * * * * * *  * * * * * * *  * *  * * * * */
-void SAMPLE_ResetCounters(void)
+void SAMPLE_ResetCounters( CFE_SB_MsgPtr_t msg )
 {
-    /* Status of commands processed by the SAMPLE App */
-    SAMPLE_HkTelemetryPkt.sample_command_count       = 0;
-    SAMPLE_HkTelemetryPkt.sample_command_error_count = 0;
+    uint16 ExpectedLength = sizeof(SAMPLE_NoArgsCmd_t);
+
+    /*
+    ** Verify command packet length
+    */
+    if (SAMPLE_VerifyCmdLength(msg, ExpectedLength))
+    {
+        Sample_AppData.CmdCounter = 0;
+        Sample_AppData.ErrCounter = 0;
+
+        CFE_EVS_SendEvent(SAMPLE_COMMANDRST_INF_EID,
+                          CFE_EVS_INFORMATION,
+		          "SAMPLE: RESET command");
+    }
 
-    CFE_EVS_SendEvent(SAMPLE_COMMANDRST_INF_EID, CFE_EVS_EventType_INFORMATION,
-		"SAMPLE: RESET command");
     return;
 
 } /* End of SAMPLE_ResetCounters() */
 
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * **/
+/*  Name:  SAMPLE_Process_CC                                                  */
+/*                                                                            */
+/*  Purpose:                                                                  */
+/*         This function Process Ground Station Command                       */
+/*                                                                            */
+/* * * * * * * * * * * * * * * * * * * * * * * *  * * * * * * *  * *  * * * * */
+void  SAMPLE_Process_CC(CFE_SB_MsgPtr_t msg)
+{
+    uint16 ExpectedLength = sizeof(SAMPLE_NoArgsCmd_t);
+    SampleTable_t *TblPtr;
+    const char *fileName = "/cf/sample_table.tbl";
+
+
+    /*
+    ** Verify command packet length
+    */
+    if (SAMPLE_VerifyCmdLength(msg, ExpectedLength))
+    {
+
+        /* Sample Use of Table */
+        CFE_TBL_GetAddress((void *)&TblPtr,
+                           Sample_AppData.TblHandles[0]);
+
+        CFE_ES_WriteToSysLog("Sample App: Table Value 1: %d  Value 2: %d", 
+                             TblPtr->Int1,
+                             TblPtr->Int2);
+    
+        Get_Crc(fileName);
+    
+    }
+
+    return;
+}
+
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * **/
 /*                                                                            */
 /* SAMPLE_VerifyCmdLength() -- Verify command packet length                   */
 /*                                                                            */
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * **/
-bool SAMPLE_VerifyCmdLength(CFE_SB_MsgPtr_t msg, uint16 ExpectedLength)
-{     
-    bool result = true;
+boolean SAMPLE_VerifyCmdLength( CFE_SB_MsgPtr_t msg, uint16 ExpectedLength )
+{
+    boolean result = TRUE;
```

Comment:
If you compile with deprecated elements removed this will fail.  Utilize bool/true/false in code.